[\#973 Issue](https://github.com/rear/rear/issues/973) `closed`: mount command for cifs is successful, but rear is showing it failed ( 0 != 0) and fails
========================================================================================================================================================

**Labels**: `support / question`, `won't fix / can't fix / obsolete`

#### <img src="https://avatars.githubusercontent.com/u/14119434?v=4" width="50">[ksquires](https://github.com/ksquires) opened issue at [2016-08-19 19:04](https://github.com/rear/rear/issues/973):

Relax-and-Recover (rear) Issue Template
=======================================

Please fill in the following items before submitting a new issue:

-   rear version (/usr/sbin/rear -V): Relax-and-Recover 1.17.2 / Git
-   OS version (cat /etc/rear/os.conf or lsb\_release -a):  
    OS\_VENDOR=RedHatEnterpriseServer  
    OS\_VERSION=7
-   rear configuration files (cat /etc/rear/site.conf or cat
    /etc/rear/local.conf):

<!-- -->

    OUTPUT=ISO
    OUTPUT_URL=cifs://192.168.1.131/backups
    BACKUP=NETFS
    BACKUP_URL=cifs://192.168.1.131/backups
    SSH_ROOT_PASSWORD="hidden"
    NETFS_KEEP_OLD_BACKUP_COPY=
    BACKUP_OPTIONS="cred=/etc/rear/.cifs"

    ++ StopIfError 'Mount command '\''mount -v -o cred=/etc/rear/.cifs //192.168.1.131/backups /tmp/rear.H4BQSEvRnMGJjOn/outputfs'\'' failed.'
    ++ ((  0 != 0  ))

the mount command works manually, but as you can see, rear is saying 0
!= 0 and calling it failed

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2016-08-20 11:02](https://github.com/rear/rear/issues/973#issuecomment-241193386):

Rename `/etc/rear/.cifs`into `/etc/rear/cifs` and retry. In 1.19 the dot
file will be supported. The documentation was saying .cifs, but it
should have been cifs

#### <img src="https://avatars.githubusercontent.com/u/14119434?v=4" width="50">[ksquires](https://github.com/ksquires) commented at [2016-08-20 15:09](https://github.com/rear/rear/issues/973#issuecomment-241205267):

so i renamed /etc/rear/.cifs to /etc/rear/cifs  
updated the following line in /etc/rear/local.conf  
BACKUP\_OPTIONS="cred=/etc/rear/cifs"

and now it seems to not recognize the cred= at all, and tries to mount
using guest:  
ERROR: Mount command 'mount -v -o rw,noatime,guest
//192.168.1.131/backups /tmp/rear.fiajS97vY00vJxr/outputfs' failed.

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2016-08-21 10:48](https://github.com/rear/rear/issues/973#issuecomment-241250689):

@ksquires please check the rear-<server>.log file - perhaps the real
reason for not using the BACKUP\_OPTIONS is shown in there?

#### <img src="https://avatars.githubusercontent.com/u/14119434?v=4" width="50">[ksquires](https://github.com/ksquires) commented at [2016-08-23 14:19](https://github.com/rear/rear/issues/973#issuecomment-241746363):

looks like the same (0!=0) issue:  
++ StopIfError 'Mount command '''mount -v -o cred=/etc/rear/cifs
//192.168.1.131/backups /tmp/rear.BOI3rMrL0vt5CmF/outputfs''' failed.'  
++ (( 0 != 0 ))

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2016-09-05 16:00](https://github.com/rear/rear/issues/973#issuecomment-244777659):

@ksquires I cannot reproduce your problem...

    ++ mount_cmd='mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs'
    ++ Log 'Mounting with '\''mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs'\'''
    ++ test 1 -gt 0
    +++ Stamp
    +++ date '+%Y-%m-%d %H:%M:%S.%N '
    ++ echo '2016-09-05 17:55:49.758768611 Mounting with '\''mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs'\'''
    2016-09-05 17:55:49.758768611 Mounting with 'mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs'
    ++ eval mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs
    +++ mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs
    domain=MYGROUP
    mount.cifs kernel mount options: ip=192.168.33.15,unc=\\server\homes,user=vagrant,,domain=MYGROUP,pass=********
    ++ StopIfError 'Mount command '\''mount -v -o cred=/etc/rear/.cifs //server/homes /tmp/rear.YNLgWyVZcvMPLDy/outputfs'\'' failed.'
    ++ ((  0 != 0  ))
    ++ AddExitTask 'umount -f -v '\''/tmp/rear.YNLgWyVZcvMPLDy/outputfs'\'' >&2'

Please investigate your log file in-depth - there must be another reason

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2016-09-05 16:01](https://github.com/rear/rear/issues/973#issuecomment-244777832):

@ksquires Perhaps remove `OUTPUT_URL=cifs://192.168.1.131/backups` as it
is the same as the BACKUP\_URL anyway?

#### <img src="https://avatars.githubusercontent.com/u/888633?u=cdaeb31efcc0048d3619651aa18dd4b76e636b21&v=4" width="50">[gdha](https://github.com/gdha) commented at [2016-09-29 09:33](https://github.com/rear/rear/issues/973#issuecomment-250417447):

No feedback received and we cannot reproduce this case - we will close
it - it can be re-opened if needed

------------------------------------------------------------------------

\[Export of Github issue for
[rear/rear](https://github.com/rear/rear).\]
